src/js/common/utils/helpers.js

```diff
@@ -125,7 +125,9 @@ export function focusElement(selectorOrElement, options) {
     : selectorOrElement;
 
   if (el) {
-    if (el.tabIndex === -1) el.setAttribute('tabindex', '-1');
+    if (el.tabIndex <= 0) {
```

Comment:
@jbalboni @1Copenut The `tabIndex` of non-tabbable elements on IE is 0. On everything else it's -1.